spec/egree/api_mappers/case_event_subscription_mapper_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+require "egree/api_mappers/case_event_subscription_mapper"
+
+module Egree
+  module ApiMappers
+    RSpec.describe CaseEventSubscriptionMapper do
+      describe ".to_api" do
+        it "creates a hash that matches the api's expected format" do
+          case_event_subscription = double("CaseEventSubscription", {
```

Comment:
Lets try and use real values as far as possible in the specs to avoid stubbing the world.